src/Uno.UI/UI/Xaml/UIElementCollectionExtensions.Xamarin.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml.Media;
+#if __ANDROID__
+using _View = Android.Views.View;
+#elif __IOS__
+using _View = UIKit.UIView;
+#elif __MACOS__
+using _View = AppKit.NSView;
+#endif
+
+namespace Windows.UI.Xaml.Controls
+{
+	public static class UIElementCollectionExtensions
+	{
+		/// <summary>
+		/// Add a native view to a <see cref="UIElementCollection"/>. It will be wrapped in a <see cref="FrameworkElement"/>-derived container to satisfy the type constraints of <see cref="UIElementCollection"/>.
+		/// </summary>
+		/// <remarks>This method is present to support collection initializer syntax for native views.</remarks>
+		public static void Add(this UIElementCollection uiElementCollection, _View view)
+		{
+			if (view is UIElement uiElement)
+			{
+				uiElementCollection.Add(uiElement);
+			}
+
+			var wrapper = VisualTreeHelper.AdaptNative(view);
+			uiElementCollection.Add(wrapper);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'uiElementCollection' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5554394)